### ISSUE TITLE
Do we need to finetune controlnet when we merge sd with different lora?

### ISSUE THREAD

**Issue by yftongbupt** [lllyasviel/ControlNet#415]
When we have trained a lora with sd and merge lora with sd to get a new base model, do we need to finetune controlnet with new base model?

**Comment by geroldmeisinger** [comment]
No. Try it out! Although I cannot explain why. see here for more info https://github.com/lllyasviel/ControlNet/discussions/12
